inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/PdfAnnoPanel.java

```diff
@@ -105,7 +105,8 @@ public void respond(AjaxRequestTarget aTarget)
             @Override
             protected void respond(AjaxRequestTarget aTarget)
             {
-                aPdfAnnotationEditor.renderPdfAnnoModel(aTarget);
+//                aPdfAnnotationEditor.renderPdfAnnoModel(aTarget);
```

Comment:
WUT? Why is the above line commented out and a message logged to system out? If you want to log a warning about an unexpected API call, better log it through the logging system. Or throw an exception... or (since you are in a AJAX call) you could even log a feedback message to the UI.

Reply:
I think I need more sleep.. this shouldn't have been commited..
This whole `AbstractDefaultAjaxBehavior` should be removed as all calls regarding annotations are now done over the `apiProvider`.